lessons/codap-bar-charts/langs/en-us/index.adoc

```diff
@@ -114,7 +120,7 @@ When time is up, invite students to share.
 [.lesson-instruction]
 * What did you discover?
 * When did the `configuration` menu appear?
-** _When there is another possible configuration of the data - for instance, when dots can be fused into bars - we see this menu._
+** _When there is another possible configuration of the data; for instance, when dots can be fused into bars, we see this menu._
 * When did the `measure` menu appear?
 ** _This menu appears when there is an opportunity to change what is shown _along with_ the points - for instance, connecting lines, a regression line, or a count_.
```

Comment:
I'm unclear whether we want to talk about regression lines when students don't know what they are and the options in the drop down menu don't mirror the term.